let a = 5;
let b = 10;
console.log(a+b);
console.log(a+b);
try{
    console.log(a+c);// error
}catch(err){
    console.log(err);
}

console.log(a+b);
console.log(a+b);